contracts/protocol/clients/voucher/BosonVoucher.sol

```diff
@@ -47,7 +47,8 @@ contract BosonVoucherBase is IBosonVoucher, BeaconClientBase, OwnableUpgradeable
     string private _contractURI;
 
     // Royalty percentage requested by seller (for all offers)
-    uint256 private _royaltyPercentage;
+    // Not used anymore. Need to stay to avoid storage shift.
+    uint256 private _royaltyPercentageUnused;
```

Comment:
should I understand there is no royaltyPercentage managed at collection level?
If so, this answers my previous question

Reply:
Yes, no royaltyPercentage is managed on the collection level, at least not in this iteration.

Maybe we could bring it back and have it this way:
A seller sets royalty recipients and percentages on the collection level. This is then the default value for all offers. If the seller wants to use it in the offer, they leave `offer.royaltyInfo` empty. But if they want different royalties for a certain offer, they populate `offer.royaltyInfo` while creating an offer.
I'm open to this approach if it makes more sense.

Anyway, even if we make collection-wide royalties, I will still move it from `bosonVoucher` to protocol level, since it's more efficient.